docker/.env

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
```

Comment:
if the docker stuff is just for tests maybe it should be in tests/docker instead